### PR TITLE
Apply an edit pass to fix several docs issues

### PR DESCRIPTION
**Pull request by ada-globus**
Includes fixes for uses of “Automate” as well as fixes for capitalization of non-proper nouns and instances where backquotes appear to have been used to indicate emphasis. This is admittedly not comprehensive, so consider this a downpayment on further docs improvements.

### CODE REVIEW

File: docs/source/authoring_flows.rst [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
 ============================================
 
-The Globus Flows Service provides users with the ability to easily define
-compositions of Actions (henceforth referred to as Flows) to perform a single,
-logical operation. Definition of such Flows requires an easy to read, author,
-and potentially visualize method of defining the Flows. For this purpose, the
+*Globus Flows* provides users with the ability to easily define
+compositions of actions (henceforth referred to as *flows*) to perform a single,
+logical operation. Definition of such flows requires an easy to read, author,
+and potentially visualize method of defining the flows. For this purpose, the
```
Comment (kurtmckee) [inline docs/source/authoring_flows.rst:9]:
```suggestion
logical operation. The method for defining such flows must be easy to read, author,
and potentially visualize. For this purpose, the
```

File: docs/source/overview.rst [hunk 2]
```diff
@@ -30,52 +29,52 @@ a specific need. Examples of these workflows might be:
...
 <https://action-provider-tools.readthedocs.io/en/latest/>`_ - a Python
 SDK that makes it easy to provide custom services that can be tied
-into the ``Globus Automate`` ecosystem of services.
+into the Globus automation ecosystem.
 
-These ``Action Providers`` form the foundation of  ``Globus Automate`` and are
-primarily used by referencing their URLs in `Flows`_.
-``Globus Automate`` allows users to flexibly piece together these individual
+These action providers form the foundation of the Globus automation ecosystem
+and are primarily used by referencing their URLs in `Flows`_.
```
[the remark this answers is not in the document]
Comment (ada-globus) [reply docs/source/overview.rst:61]: According to docutils refs should be case-insensitive. I tested locally and it appeared to do the right thing in the rendered HTML, so I'm going to apply!

**Comment by ada-globus** [comment]
Thanks @kurtmckee, great catches!

**Review by kurtmckee — APPROVED** [review]
I wasn't able to review with a fine-toothed comb but I think this is a significant improvement!

**Pull request merged**